test/lifecycle/Pausable.test.js

```diff
@@ -64,4 +64,12 @@ contract('Pausable', function () {
     const drasticMeasureTaken = await this.Pausable.drasticMeasureTaken();
     drasticMeasureTaken.should.be.false;
   });
+
+  it('should log events appropriately', async function () {
+    const setPauseLogs = (await this.Pausable.pause()).logs;
+    setPauseLogs.find(e => e.event === 'Pause');
+
+    const setUnPauseLogs = (await this.Pausable.unpause()).logs;
+    setUnPauseLogs.find(e => e.event === 'UnPause');
```

Comment:
I didn't notice this typo... This must be a test's bug.. I will investigate

Reply:
I just verified that with any string as the event name, the test was passing.. this was because *find* is returning *undefined* if the event is not found, so I added an extra check to make sure that the event is actually in the set, if not, it asserts.